src/pages/Home.jsx

```diff
@@ -28,25 +29,49 @@ const Home = () => {
   const isReallySmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
   const iconSize = isReallySmallScreen ? 'medium' : 'large';
 
+  // Logo section elements
+  const heading = 'PASS';
+  const punctuation = (
+    <Box component="span" sx={visuallyHidden}>
+      :
+    </Box>
+  );
+  const subheading = 'Personal Access System for Services';
+
   const logoSection = isReallySmallScreen ? (
-    <Stack justifyContent="center" alignItems="center" spacing={2} mb={2}>
-      <Typography variant="h1" fontWeight="500" fontSize="72px" color="primary">
-        PASS
-      </Typography>
-      <Box component="img" src="/assets/PASSLogolightmode.png" alt="PASS logo" width="50%" />
-    </Stack>
+    <Typography component="h1" color="primary">
+      <Stack component="span" justifyContent="center" alignItems="center" spacing={2} mb={2}>
+        <Typography variant="h1" component="span" fontWeight="500" fontSize="72px">
+          {heading}
+          {punctuation}
+        </Typography>
+        <Box component="img" src="/assets/PASSLogolightmode.png" alt="" width="50%" />
```

Comment:
Oh yeah, got some clarity from my coworkers about alt text for logos. You are correct that in this case, it is decorative so it should be left empty. Sorry about all the confusion with that 🙇